pyaedt/hfss.py

```diff
@@ -713,7 +714,7 @@ def create_discrete_sweep(self, setupname, sweepname="SinglePoint", freq="1GHz",
                 sweepdata.props["Type"] = "Discrete"
                 sweepdata.props["RangeType"] = "SinglePoints"
                 sweepdata.update()
-                self._messenger.add_info_message("Sweep Created Correctly")
+                self._messenger.add_info_message("Sweep was created correctly.")
```

Comment:
@PipKat This edit needs to be updated in the docstring example for the function since this doctest is failing. On line 690, you can change it to be `pyaedt Info: Sweep was created correctly.`